smoke/tests/commit_test.go

```diff
@@ -0,0 +1,131 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/dragonflyoss/nydus/smoke/tests/tool"
+	"github.com/dragonflyoss/nydus/smoke/tests/tool/test"
+	"github.com/google/uuid"
+)
+
+type CommitTestSuite struct {
+	t                          *testing.T
+	testImage                  string
+	testCommittedImage         string
+	testContainerName          string
+	testCommittedContainerName string
+	snapshotter                string
+}
+
+func (c *CommitTestSuite) TestCommitContainer() {
+	ctx := tool.DefaultContext(c.t)
+	c.snapshotter = os.Getenv("SNAPSHOTTER")
```

Comment:
We'd better use `scenarios := tool.DescartesIterator{}` to manage parameters. :)